frontend/dockerfile/dockerfile_test.go

```diff
@@ -252,27 +255,32 @@ func init() {
 func TestIntegration(t *testing.T) {
 	integration.Run(t, allTests, opts...)
 
-	integration.Run(t, securityTests, append(append(opts, securityOpts...),
-		integration.WithMatrix("security.insecure", map[string]interface{}{
-			"granted": securityInsecureGranted,
-			"denied":  securityInsecureDenied,
-		}))...)
-	integration.Run(t, networkTests, append(opts,
-		integration.WithMatrix("network.host", map[string]interface{}{
-			"granted": networkHostGranted,
-			"denied":  networkHostDenied,
-		}))...)
 	integration.Run(t, lintTests, opts...)
 	integration.Run(t, heredocTests, opts...)
 	integration.Run(t, outlineTests, opts...)
 	integration.Run(t, targetsTests, opts...)
 
-	integration.Run(t, reproTests, append(opts,
-		// Only use the amd64 digest,  regardless to the host platform
-		integration.WithMirroredImages(map[string]string{
-			"amd64/bullseye-20230109-slim:latest": "docker.io/amd64/debian:bullseye-20230109-slim@sha256:1acb06a0c31fb467eb8327ad361f1091ab265e0bf26d452dea45dcb0c0ea5e75",
-		}),
-	)...)
+	func(t *testing.T) {
```

Comment:
Why do you need the closure here?

Reply:
Since I needed to skip the whole block when running on Windows, see L264...

Reply:
I don't understand. What's the difference with this code, with and without closure.

Reply:
I see, no difference, was just a logical grouping. Fixing.